staging/src/k8s.io/apimachinery/pkg/util/httpstream/spdy/connection_test.go

```diff
@@ -247,12 +247,12 @@ func TestConnectionPings(t *testing.T) {
 		t.Fatalf("client: error connecting to proxy: %v", err)
 	}
 	defer clConn.Close()
-	start := time.Now()
 	clSPDYConn, err := NewClientConnection(clConn)
 	if err != nil {
 		t.Fatalf("client: error creating spdy connection: %v", err)
 	}
 	defer clSPDYConn.Close()
+	start := time.Now()
```

Comment:
see line 278, it uses this time to count the number of pings, but if the connection takes more than 3*pingPeriod to be estsablished, then this test will fail